packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -964,12 +959,11 @@ def response_headers_and_deserialization(
     def handle_error_response(self, builder: OperationType) -> List[str]:
         async_await = "await " if self.async_mode else ""
         retval = [f"if response.status_code not in {str(builder.success_status_codes)}:"]
-        retval.extend(
-            [
-                "    if _stream:",
-                f"        {async_await} response.read()  # Load the body in memory and close the socket",
-            ]
-        )
+        response_read = f"    {async_await}response.read()  # Load the body in memory and close the socket"
```

Comment:
we can always call read, regardless of stream or not

Reply:
It is very strange to judge _stream when its value is `True`, so I think it is better to keep the logic.